tensor2tensor/models/transformer.py

```diff
@@ -802,7 +803,7 @@ def transformer_base_v1():
   hparams.initializer = "uniform_unit_scaling"
   hparams.weight_decay = 0.0
   hparams.optimizer_adam_beta1 = 0.9
-  hparams.optimizer_adam_beta2 = 0.98
+  hparams.optimizer_adam_beta2 = 0.997
```

Comment:
Why changing it here in `_v1`? I thought the whole point of having `_v1` and `_v2` is to keep replicability (on best-effort basis).

Reply:
Yes, agreed. Reverting. Thanks for catching it. It must have slipped through in review.